astro.config.mjs

```diff
@@ -19,8 +19,36 @@ export default defineConfig({
           autogenerate: { directory: 'overview' },
         },
         {
-          label: 'How to Invest using Angor',
-          autogenerate: { directory: 'how-to-investsing Angor' },
+          label: 'getting started',
+          autogenerate: { directory: 'creating a wallet' },
```

Comment:
Use - instead of space for directory name

Reply:
Noted